json_to_csv.py

```diff
@@ -0,0 +1,46 @@
+import os
+import json
+import csv
+
+
+def fun(in_dir,out_dir,**d):
+
+    os.chdir("C:/Users/anurag/Downloads/darkflow-master/darkflow-master") #change directory to that path where object detector YOLO v2 resides.
+    if not os.path.exists(in_dir):
+        print("INVALID DIRECTORY ENTERED")
+    else:
+        os.system("python flow --imgdir "+in_dir+"/ --model cfg/yolo.cfg --load bin/yolo.weights --json")
+
+        filelst = os.listdir(in_dir+"/out")
+
+        if not os.path.exists(out_dir):
+            os.mkdir(out_dir)
+
+        for j in filelst:
+            infile = open(os.path.join(in_dir+"/out", j), "r")  # input the json file directory
+            data = json.loads(infile.read())

```

Comment:
`infile` is not closed. That's why always prefer 

``` 
       with open(in_filepath) as infile: 
          for line in infile:
             # Sample line: xxxx
             data = json.loads(line) ```

Reply:
Ok, changes implemented successfully